packages/eslint-config-pwa-kit/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "eslint-config-pwa-kit",
+  "version": "2.0.0-dev.4",
+  "description": "ESLint config used by PWA Kit",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/SalesforceCommerceCloud/pwa-kit.git"
+  },
+  "license": "SEE LICENSE IN LICENSE",
+  "author": "dev@mobify.com",
+  "main": "index.js",
+  "scripts": {
+    "lint": "npm run lint:js",
+    "lint:fix": "npm run lint:js -- --fix",
+    "lint:js": "internal-lib-build lint ."
+  },
+  "dependencies": {
+    "eslint": "7.31.0",
+    "eslint-config-prettier": "4.0.0",
+    "eslint-import-resolver-webpack": "0.10.0",
+    "eslint-plugin-header": "^3.1.1",
```

Comment:
Not sure that `eslint-plugin-header` should be here. The header rule is defined at the [monorepo root](https://github.com/SalesforceCommerceCloud/pwa-kit/blob/120dfb431f03fd8811c337cc70c2898a84a98eb6/.eslintrc.js), and is specifically for the monorepo, not generated projects. I _think_ the headers should be enforced as desired if we install this plugin at the monorepo root, but I haven't verified.